src/pull-request-poller/pull-request-poller.tsx

```diff
@@ -30,6 +31,17 @@ export default class PullRequestPoller extends React.Component<FlagToListenTo, {
 							const pullrequests = doc.querySelectorAll('a[id^=\'issue_\']');
 							for (var i = 0, element; element = pullrequests[i]; i++) {
 								let newPr = pullrequests[i].innerHTML;
+								let author = pullrequests[i].parentElement.querySelector('a.muted-link').innerHTML;
+								if (this.props.authors.length > 0) {
+									if (this.props.ExcludeOrOnly && !this.props.authors.includes(author)) {
+										//doesn't show the notification if it's not matching only rules
+										continue;
+									}
+									if (!this.props.ExcludeOrOnly && this.props.authors.includes(author)) {
+										//doesn't show the notification if author is excluded
+										continue;
+									}
```

Comment:
Consider XOR: `if (this.props.ExcludeOrOnly ^ this.props.authors.includes(author)) {` to consolidate both `if`s, but keep both comments.
Also, I tend to write comments as proper sentences with capitals and everything, including a space after the `//`, and reserve writing `//` not followed by a space for commented out code, so I can read 1 char and tell which is which.  Just a suggestion.

Reply:
XOR maybe a bit more difficult to read unfortunately :/

Reply:
If everyone says that then no one learns xor :-p